src/model/services/mapi/authorizationv1/types.ts

```diff
@@ -18,7 +18,7 @@ export interface IResourceRule {
   //  "*/foo" represents the subresource 'foo' for all resources in the specified apiGroups.
   resources: string[];
   // ResourceNames is an optional white list of names that the rule applies to.  An empty set means that everything is allowed.  "*" means all.
```

Comment:
```suggestion
  // ResourceNames is an optional list of allowed names that the rule applies to.  An empty set means that everything is allowed.  "*" means all.
```

We'll try to avoid the terms "white list" and "black list" in the product. Might as well avoid it in code.